qemu/tcg/tcg.c

```diff
@@ -3728,7 +3728,7 @@ int tcg_gen_code(TCGContext *s, TranslationBlock *tb)
     TCGOp *op;
 
 #ifndef NDEBUG
-    if (getenv("UNICORN_DEBUG")) {
+    if (is_log_level_active(CPU_LOG_TB_IN_ASM)) {
```

Comment:
is `CPU_LOG_TB_IN_ASM` really defined somewhere?

Reply:
Yes it is defined in the "log.h" file, where I have implemented the logging. 
If it's not defined, a compiler error would be generated.

Not sure if that is the right log level bit mask, as it's very very verbose. Feel free to change it, but I thougt it fits the most.

In the future, we might want to add Unicorn specific log levels (and macros) to distingiush between qemu und unicorn logs. (Just a quick popped up idea)

Reply:
Ohhh yes, github folded the definitions.